src/app/debug/table/table.component.css

```diff
@@ -79,19 +78,14 @@ th {
 
 .progress-report-timout {
```

Comment:
```suggestion
.progress-report-timeout {
```
Small typo. See below for reference in table template

Reply:
fixed